lib/tasks/deal/publish_and_unpublish.rake

```diff
@@ -0,0 +1,20 @@
+namespace :deal do
```

Comment:
rename file to something more descriptive

Reply:
renamed to `update_live_status`